tensorflow/core/kernels/crop_resize_bilinear_core.h

```diff
@@ -4962,11 +4815,12 @@ void CropResizeCastImage<T, U>::ResizeRow_load8_4ch_(
     __m128 top3 = _mm_add_ps(tl3, _mm_mul_ps(x_lerp, _mm_sub_ps(tr3, tl3)));
     __m128 bot3 = _mm_add_ps(bl3, _mm_mul_ps(x_lerp, _mm_sub_ps(br3, bl3)));
 #endif
-    res_[0] = _mm_add_ps(top0, _mm_mul_ps(y_lerp, _mm_sub_ps(bot0, top0)));
-    res_[1] = _mm_add_ps(top1, _mm_mul_ps(y_lerp, _mm_sub_ps(bot1, top1)));
-    res_[2] = _mm_add_ps(top2, _mm_mul_ps(y_lerp, _mm_sub_ps(bot2, top2)));
-    res_[3] = _mm_add_ps(top3, _mm_mul_ps(y_lerp, _mm_sub_ps(bot3, top3)));
-    this->write_4ch(ysA_output_ptr + load8_x_[current] * CHANNELS, res_);
+    __m128 res[4];
+    res[0] = _mm_add_ps(top0, _mm_mul_ps(y_lerp, _mm_sub_ps(bot0, top0)));
```

Comment:
Thanks for incorporating the feedback. My earlier comments about explicit FMA also applies for the sequence of instructions seen here as well and _mm_fmadd_ps() would be beneficial..

Reply:
I have updated the code according to your advice. Thanks, Thor